README.md

```diff
@@ -1,258 +1,337 @@
-Mimic, the Mycroft TTS Engine
-==============================
+#Mimic - The Mycroft TTS Engine
```

Comment:
These are the lines I talked about. The 
```==============================```

creates a horizontal line that the `#...` doesn't generate.